samples/unit-tests/series-pie/gradient/demo.js

```diff
@@ -1,4 +1,4 @@
-QUnit.test('Pie radial gradient', function (assert) {
+QUnit.test('Pie radial gradient (#3801)', function (assert) {
```

Comment:
This tests makes other, downstream tests fail because it modifies the default colors through `Highcharts.setOptions`. The fix is to store the original colors array first, then reset it at the very end of the test.

```js
const colors = Highcharts.defaultOptions.colors;

// The test body here

// Reset
Highcharts.setOptions({
    colors
});
```